frontend/src/Student/components/VideoSubmissionForm.js

```diff
@@ -187,9 +203,27 @@ class VideoSubmissionForm extends Component {
                           name='submittingAsGroup'
                           value='no'
                           required
+                          disabled={!canSubmitAsSingle}
                           checked={values.submittingAsGroup === 'no'}
                         />
-                        <span className='ml-2'>No</span>
+                        {canSubmitAsSingle ? (
```

Comment:
Can we pull the tooltip out into a higher level component so we're not doing the same thing in three places?